wiki/en/Running-a-Server.md

```diff
@@ -125,185 +144,174 @@ To register with one of the Directories built into the Jamulus Client, replace `
 
-#### Viewing The Logs
+`[name];[city];[country as two-letter ISO country code]`
 
-Jamulus will log to the system log file if you left the `StandardOutput=journal` setting in the unit file.
+See [two-letter ISO country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements)
 
-To view the log, use `journalctl` (to exit press Ctrl-C). For example, to read the system log file, filtered for the Jamulus service:
+Note that semicolon and newline characters are not allowed in `name` and `city` values.
 
-`journalctl -f -u jamulus-headless`
+##### `-L or --licence` 
+Show an agreement window before users can connect
 
-#### Controlling Recording
+##### `-w or --welcomemessage` 
+Welcome message on connect. Can be given as a string or filename, and can contain HTML.
 
-When using the recording function with the `-R` command line option, if the Server receives a SIGUSR1 signal during a recording, it will start a new recording in a new Directory. SIGUSR2 will recording enabled on/off.
+##### `--serverpublicip` 
+The public IP address of the Server if connecting to a Directory behind the same NAT. See [the Directories guide](Directories) for further information.
 
-To send these signals using systemd, create the following two `.service` files in `/etc/systemd/system`, calling them something appropriate (e.g. `jamulusTogglerec.service`).
+##### `--directoryfile` 
+_Directories only:_ Remember registered Servers even if the Directory is restarted. See [the Directories guide](Directories) for further information.
 
-**Note:** You will need to save recordings to a path _outside_ of the jamulus home Directory, or remove `ProtectHome=true` from your systemd unit file (be aware that doing so is however a potential security risk).
+##### `-f or --listfilter` 
+_Directories only:_ Whitelist Servers registering on the Server. See [the Directories guide](Directories) for further information.
 
-For turning recording on or off (depending on the current state):
-
-~~~
- [Unit]
- Description=Toggle recording state of Jamulus Server
- Requisite=Jamulus-Server
+---
 
- [Service]
- Type=oneshot
- ExecStart=/bin/systemctl kill -s SIGUSR2 Jamulus-Server
-~~~
+### General Server options
+These options can be used regardless of which mode your server is running in (although some may not be relevant for Directories).
 
-For starting a new recording:
+##### `-d or --discononquit` 
+Normally, when a Server is stopped or restarted, any Clients that have not used their “Disconnect” buttons will re-establish connection when the Server comes back up again. Using this option forces Clients to manually re-establish their connections to the Server.
 
-~~~
- [Unit]
- Description=Start a new recording on Jamulus Server
- Requisite=Jamulus-Server
+##### `-F or --fastupdate` 
+Reduces latency if Clients connect with “Enable Small Network Buffers” option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.
```

Comment:
```suggestion
Reduces latency if Clients connect with **Enable Small Network Buffers** option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.
```